// Copyright 2015 The Chromium OS Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
syntax = "proto2";
option optimize_for = LITE_RUNTIME;

package soma;

import "container_spec.proto";

service Soma {
  rpc GetContainerSpec (GetContainerSpecRequest) returns (GetContainerSpecResponse);
}

message GetContainerSpecRequest {
  optional string service_name = 1;
}

message GetContainerSpecResponse {
  optional ContainerSpec container_spec = 1;
}
